spec/02-integration/03-admin_api/03-consumers_routes_spec.lua

```diff
@@ -23,7 +24,7 @@ describe("Admin API", function()
   before_each(function()
     helpers.dao:truncate_tables()
     consumer = assert(helpers.dao.consumers:insert {
-      username = "bob",
+      username = "bob pop",
```

Comment:
It is worth inserting another consumer with such a space in its name and creating a new test for "retrieving by username with percent-encoded characters", because those two tests prove different behaviors.